main/kotlin/com/example/people_managment_system/dtos/EmployeeDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.people_managment_system.dtos
+
+import java.io.Serializable
+
+data class EmployeeDto(

```

Comment:
No need to have the `Dto` name in the class as we can understand what it is by seeing it is a `data class`.

Reply:
There is another class called Employee that corresponds to employee entities in the 'employees' table in the Postgres database so I left this name as it was.  See if you think this was correct when you see the updated code.